monkey/monkey_island/cc/services/attack/mitre_api_interface.py

```diff
@@ -1,6 +1,6 @@
 from typing import List, Dict
 
-from stix2 import FileSystemSource, Filter, CourseOfAction, AttackPattern, v20
```

Comment:
We can rollback these changes since this is was not the problem

Reply:
I actually prefer to keep it this way, since `v20._*` feels like a pretty internal-y thing